src/Trakx.Circle.ApiClient/openApi3.yaml

```diff
@@ -6,9 +6,6 @@ info:
     name: trakx.io
     email: support@trakx.io
     url: https://trakx.io
-servers:
```

Comment:
If we keep this, the NSwag will generate the code with this url, instead of reading from configuration class... 😞